modules/cms-kit/src/Volo.CmsKit.Admin.Application.Contracts/Volo/CmsKit/Admin/Blogs/IBlogAdminAppService.cs

```diff
@@ -1,8 +1,13 @@
 ﻿using System;
+using System.Threading.Tasks;
+using Volo.Abp.Application.Dtos;
 using Volo.Abp.Application.Services;
 
 namespace Volo.CmsKit.Admin.Blogs;
 
 public interface IBlogAdminAppService : ICrudAppService<BlogDto, Guid, BlogGetListInput, CreateBlogDto, UpdateBlogDto>
 {
+    Task<ListResultDto<BlogDto>> GetAllListAsync();
+    
+    Task MoveAllBlogPostsAsync(Guid blogId, Guid? assignToBlogId);
```

Comment:
It would be better to set the nullable parameter as null as default, so we don't explicitly pass it.

```suggestion
    Task MoveAllBlogPostsAsync(Guid blogId, Guid? assignToBlogId = null);
```